delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/directives/ResponseToJsonLd.scala

```diff
@@ -106,12 +106,9 @@ object ResponseToJsonLd extends FileBytesInstances {
       override def apply(statusOverride: Option[StatusCode]): Route = {
         val flattened = io.flatMap { fr => fr.content.attempt.map(_.map { s => fr.metadata -> s }) }.attempt
         onSuccess(flattened.runToFuture) {
-          case Left(complete: Complete[E]) => emit(complete)
-          case Left(reject: Reject[E])     => emit(reject)
-          case Right(Left(c))              =>
-            implicit val valueEncoder = c.value.encoder
-            emit(c.value.value)
-
+          case Left(complete: Complete[E])       => emit(complete)
```

Comment:
This file is currently duplicated for the Cats migration, we need to keep the two of them in sync

Reply:
Can I port the FileRoutes across to the cats? I don't feel great about this being untested

Reply:
Perhaps now I'm unit test this class, I can test both